src/app/core/services/misc/log.service.ts

```diff
@@ -36,14 +36,16 @@ export class LogService {
       return (<Array<any>>obj).map(o => this.formatObject(o)).toString()
     } else if (typeof obj !== 'object') {
       return String(obj)
-    } else if (obj.toString !== Object.prototype.toString) {
+    } else if (obj.toString() !== Object.prototype.toString()) {
```

Comment:
The old notation was intentional, why was this change made? The intention is that if an object defines a unique string as its representation, it should be used in the console. This is to prevent `[Object object]` logs and to allow `MyObj{custom string}` logs.

Reply:
Yeah I made the change because they were returning the function definition instead of `[Object object]`. Returning `[Object object]` was your intention right? Sorry if I'm getting it confused.

Reply:
Yes, the function definition was the intention: if the prototype overrides the toString method (so changes the definition), use the updated definition.

Reply:
Oh ok sorry about that. I got it confused.